### PR TITLE
Update azure.md

### PR DESCRIPTION
**Pull request by athuls1992**
Added complete steps to resolve persistent storage issue when deploying in Azure container @chungyau97 @HenryHengZJ

### CODE REVIEW

File: deployment/azure.md [hunk 1]
```diff
@@ -68,3 +68,26 @@ az containerapp create \
 ```
 
 4. Visit the url printed from the output of the previous command.
+
+##Persistent Disk in Container Apps
```
Comment (meaboutsoftware) [inline deployment/azure.md:72]: I am voting to not to accept this solution as it is unstable due to the nature of sqlite (that is designed to not to be a database for concurrent users) and poor integration with File Share. It sometimes saves the data, sometimes not, sometimes is able to read and sometimes not.

I am now testing the solution with a file share as store (without sqllite), similar as it is done in Render.

File: deployment/azure.md [hunk 2]
```diff
@@ -68,3 +68,26 @@ az containerapp create \
...
+1. Create a Storage Account in Azure portal
+2. In Storage Account create a new FileShare with Read/Write access
+3. Connect FileShare through Azure Files (tab on the left -> Settings/Azure Files) in Container Apps Environment
+4. Go to the Container instance (Under the revision select containers) and create a new revision.
+5. Within the new revision select the container settings edit & redeploy option, Navigate to volume mounts.
+6. Select File Shares and from dropdown chose the created fileshare (which is created in Step 2).
+7. Provide a mount path to selected fileshare(For eg. "/root/.flowise")
+8. After setting the fileshare. Select the basics tab, modify the existing command override to:
+/bin/sh,-c,apk add sqlite && sqlite3 /root/.flowise/database.sqlite 'PRAGMA journal_mode=WAL;' && flowise start
+
```
Comment (meaboutsoftware) [inline deployment/azure.md:87]: I am lacking the step to configure an environment variable for DATABASE_PATH

**Review by meaboutsoftware — CHANGES_REQUESTED** [review]

**Review by meaboutsoftware — CHANGES_REQUESTED** [review]
I would wait with description of this solution due to instability. It does not work as expected - sometimes read existing chatflows, sometimes not, sometimes saves, sometimes not.

**Comment by meaboutsoftware** [comment]
I vote to close this PR without merging. Look at PR #10.

**Comment by HenryHengZJ** [comment]
closing as #10 merged

**Pull request closed without merging**